api/src/main/java/org/apache/cloudstack/api/command/user/template/CreateTemplateCmd.java

```diff
@@ -251,26 +267,33 @@ public long getEntityOwnerId() {
             }
         }
 
-        if(projectId != null){
+        Account accountToUse = callingAccount;
+        if (accountName != null && domainId != null) {
+            try {
+                accountToUse = _accountService.getAccount(_accountService.finalyzeAccountId(accountName, domainId, projectId, true));
+
+            } catch (Exception exception) {
+                s_logger.info("Unable to find accountId associated with accountName=" + accountName + " and domainId="
```

Comment:
If it is an exception better to log as error or warn depending on the severity

Reply:
Modified info to warn, included debug log as well to capture exception in log.